### PR TITLE
Use gubernator.k8s.io and testgrid.k8s.io instead of the appspot.com variants

### PR DESCRIPTION
**Pull request by ixdy**
Also, fix a small bug in which gubernator was not requiring session cookies to be sent over HTTPS. (I'm not sure how much this matters, though, as gubernator is already configured to require HTTPS.)

### CODE REVIEW

File: testgrid/README.md [hunk 1]
```diff
@@ -272,13 +272,13 @@ TestGrid assigns dashboard tabs a status based on recent test runs.
 You can get a small widget showing the status of your dashboard tab, based on
 the tab statuses above! For example:
 
-`sig-testing-misc#bazel`: [![sig-testing-misc/bazel](https://amerai-imagefix-dot-k8s-testgrid.appspot.com/q/summary/sig-testing-misc/bazel/tests_status?style=svg)](https://k8s-testgrid.appspot.com/sig-testing-misc/bazel)
+`sig-testing-misc#bazel`: [![sig-testing-misc/bazel](https://amerai-imagefix-dot-k8s-testgrid.appspot.com/q/summary/sig-testing-misc/bazel/tests_status?style=svg)](https://testgrid.k8s.io/sig-testing-misc/bazel)
```
Comment (ixdy) [inline testgrid/README.md]: @michelle192837 should this still be using amerai-imagefix-dot-k8s-testgrid.appspot.com? or is the live version working now?

Comment (michelle192837) [reply testgrid/README.md]: Live version is good!

**Comment by michelle192837** [comment]
Let me know if you need a new version of TestGrid pushed first?

File: testgrid/README.md (hunk 1 above)
Comment (ixdy) [reply testgrid/README.md]: fixed, thanks!

**Comment by ixdy** [comment]
@michelle192837 I think the only change was to require https on testgrid, which is nice-to-have, but not a requirement, so that can be pushed whenever convenient.

**Comment by michelle192837** [comment]
Cool. If it's not a requirement then a build with that change will probably go live later this week.

**Comment by ixdy** [comment]
cc @amwat 

This probably won't break anything.

I'm not sure how many things I'll need to manually deploy with these changes.

/hold cancel

**Pull request merged**